test/index.test.ts

```diff
@@ -59,10 +59,21 @@ afterAll(async () => {
 const commands = {
   vanilla: [projectName, ENTER, ENTER, ENTER, ENTER, ENTER],
 };
+const flags = {
+  vanilla: [projectName, '-t', 'vanilla', '-l', 'id', '--js', '--use-npm'],
+};
 
 describe('create-liff-app', () => {
   it('files properly created', async () => {
-    const result = await installProject(commands.vanilla);
+    const result = await installProject({ inputs: commands.vanilla });
+    const templateFiles = fs.readdirSync(templatePath).map(f => rename[f] ? rename[f] : f);
+    const expectedFiles = templateFiles.concat(generatedFiles);
+    expect(result.files.sort()).toEqual(expectedFiles.sort());
+    expect(result.exitCode).toBe(0);
+  });
+
+  it('creates app using only flags', async () => {
```

Comment:
If we're adding this here, we need to change to use `beforeEach` instead of `beforeAll` I think?